inspire_disambiguation/core/helpers.py

```diff
@@ -320,6 +322,21 @@ def process_clustering_output(clusterer):
     return output
 
 
+def train_validation_split(curated_signatures, split_fraction):
+    """
+    Returns dictionary of training and test Signatures
+    """
+    seed(99)
+    curated_signatures_dict = load_signatures(curated_signatures)
+    train_signatures_uuids = set(sample(curated_signatures_dict.keys(),
+                                 k=round(len(curated_signatures_dict) * split_fraction)))
+    train_signatures = {k: curated_signatures_dict.get(k) for k in train_signatures_uuids}
```

Comment:
do you need to do `.get(k)` here? wouldn't `[k]` work? also I would call this `uuid` instead of `k`.